p2p/conn/connection.go

```diff
@@ -708,9 +710,9 @@ func (c *MConnection) Status() ConnectionStatus {
 		status.Channels[i] = ChannelStatus{
 			ID:                channel.desc.ID,
 			SendQueueCapacity: cap(channel.sendQueue),
-			SendQueueSize:     int(atomic.LoadInt32(&channel.sendQueueSize)),
+			SendQueueSize:     int(atomic.LoadInt32(&channel.sendQueueSize)), //nolint:gosec
 			Priority:          channel.desc.Priority,
-			RecentlySent:      atomic.LoadInt64(&channel.recentlySent),
+			RecentlySent:      atomic.LoadInt64(&channel.recentlySent), //nolint:gosec
```

Comment:
I had to manually skip the `gosec` linter checks here, otherwise it complains:

```
G601: Implicit memory aliasing in for loop.
```

My understanding of how this code works is that we're immediately loading the int values from the pointers during the current iteration, so it's irrelevant. Please correct me if I'm wrong though.

Reply:
I think it is complaining about the loop variable (potentially) being rewritten while using it as a reference. IICR @jmalicevic fixed this